examples/webgl_materials_cubemap_dynamic.html

```diff
@@ -73,19 +73,23 @@
 
 				//
 
-				cubeRenderTarget1 = new THREE.WebGLCubeRenderTarget( 256 );
+				const envSize = 64; // minimum size for roughness >= 0.1
```

Comment:
@Mugen87 @elalish I think it would be better, as an example, to do this:

```js
const envSize = 256; // the minimum size if roughness >= 0.1 is 64
```

Are the default `WebGLCubeRenderTarget` options the ones you want?

Reply:
I guess 64 was used so we have at least one use case with a small sized environment map. TBH, I'm not sure which value is better. Both setups makes sense.

Reply:
Since users copy these patterns, I suggest we set it to a recommended size -- not to an edge case for testing.

Reply:
Actually, I changed in here on purpose. Yes, this did catch an error, but more importantly, there's nothing special about 256. We've actually found most of our users preferring smaller envMaps to get reasonable HDR downloads (since they're PNGs internally). 256 is required for roughness >= 0.05, while 64 is for roughness >= 0.1; I consider that to be a very worthwhile trade in performance (both GPU cache and download) for the vast majority of users.

Reply:
I think it would be good to document this.

@elalish what are the roughness ranges for 32, 128 and 512?

| envmap size | roughness |
| --- | --- |
| 32 | ??? |
| 64 | >= 0.1 |
| 128 | ??? |
| 256 | >= 0.05 |
| 512 | ??? |

Once we complete the table we could add it to the docs:
https://threejs.org/docs/?q=pmrem#api/en/extras/PMREMGenerator

Reply:
`roughness = ( 2 ^ ( -0.5 * mip ) ) / 1.16`, where dimension is `2 ^ mip`. So: 

- 16: mip 4: roughness 0.21
- 32: mip 5: roughness 0.15
- 64: mip 6: roughness 0.11
- 128: mip 7: roughness 0.076
- 256: mip 8: roughness 0.054
- 512: mip 9: roughness 0.038
- 1024: mip 10: roughness 0.027